src/azure-cli/azure/cli/command_modules/vm/_params.py

```diff
@@ -500,6 +500,13 @@ def load_arguments(self, _):
 
         c.argument('protect_from_scale_in', arg_type=protection_policy_type, help="Protect the VM instance from scale-in operations.")
         c.argument('protect_from_scale_set_actions', arg_type=protection_policy_type, help="Protect the VM instance from scale set actions (including scale-in).")
+        c.argument('enable_terminate_notification', min_api='2019-03-01', arg_type=get_three_state_flag(),
+                   help='Enable terminate notification')
+
+    for scope in ['vmss create', 'vmss update']:
+        with self.argument_context(scope) as c:
+            c.argument('terminate_notification_time', min_api='2019-03-01',
```

Comment:
Can you provide default value for your argument?

Reply:
> 
> 
> Can you provide default value for your argument?

The expected default behaviour is disabled. When users don't set this argument, it is disabled.